src/components/editor/InputListLOC.jsx

```diff
@@ -145,4 +141,4 @@ const mapDispatchtoProps = dispatch => ({
   }
 })
 
-export default connect(mapStatetoProps, mapDispatchtoProps)(InputListLOC)
\ No newline at end of file
+export default connect(mapStatetoProps, mapDispatchtoProps)(InputListLOC)
```

Comment:
whitespace gaff?  (does the linter care about this?  if not, I sure don't)

Reply:
I think this is an artifact of Atom